tests/test_datasets/parameters.prm

```diff
@@ -0,0 +1,119 @@
+
+[Original parameters]
+# This section describes the radial distributions of the parameters "chord", "pitch", "rake", "skew", and "camber"
+# at given radial sections.
+
+Radial distributions of parameters:
+# ##############################################################################################################################
+# RADII               |CHORD               |PITCH               |RAKE                |SKEW                |CAMBER              #
```

Comment:
divide it in 6 different parameters, otherwise it is a pain to compile it by hand.